tests/phpunit/tests/actions.php

```diff
@@ -530,6 +596,9 @@ function apply_testing_filter() {
 		$this->assertFalse( doing_filter( 'testing_nested' ) );
 	}
 
+	/**
+	 * @covers ::doing_filter
+	 */
```

Comment:
```suggestion
```
This is a callback method used by test `test_doing_filter_real`. The `@covers` should be applied to the test itself.